### PR TITLE
Fix missing parameter value in create challenge controller function

### PR DESCRIPTION
**Pull request by tschaffter**
This PR fixes the issue reported in [this comment](https://github.com/Sage-Bionetworks/rocc-portal/pull/24#issuecomment-742137970).

This PR started investigating another issue related to date. My use of `timestamp()` is WIP and may be removed when continuing working on this date issue. Another issue I would is relative to saving `Challenge.organizations` and `Challenge.organizers`.

### CODE REVIEW

**Comment by vpchung** [comment]
Ah, okay.  The reason I had switched from `DateTimeField` to `StringField` was because some challenges only have the year for the start and/or end date. Would having just a year be an issue if we move back to datetime?

File: server/openapi_server/dbmodels/challenge.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 class Challenge(BaseDocument):
     name = StringField(required=True, unique=True)
-    startDate = StringField(required=True, regex=r"^\d{4}(-\d{2})?(-\d{2})?$")
-    endDate = StringField(required=True, regex=r"^\d{4}(-\d{2})?(-\d{2})?$")
+    startDate = DateTimeField(required=True)
```
Comment (vpchung) [inline server/openapi_server/dbmodels/challenge.py:10]: Refer to PR comment.

Comment (tschaffter) [reply server/openapi_server/dbmodels/challenge.py:10]: >Would having just a year be an issue if we move back to datetime?

Comment (tschaffter) [reply server/openapi_server/dbmodels/challenge.py:10]: How many past challenges do not have day-month-year information?

Comment (tschaffter) [reply server/openapi_server/dbmodels/challenge.py:10]: Most of the content in ROCC will come from future challenges, so they have more weights than past challenges. I would prefer to try getting more accurate information from past challenges than using this lack of information to make a design decision. If we can't find more information about a challenge, we could do 01-01-YYYY (I tried saving 00-00-YYYY in MongoDB for DateTime objects and it doesn't work).

Comment (vpchung) [reply server/openapi_server/dbmodels/challenge.py:10]: Luckily, there are only 3 dates that don't have a month and day. I'll see if I can revisit those challenges to get a more descriptive date.

**Pull request merged**